ethcore/types/src/engines/machine.rs

```diff
@@ -44,8 +44,6 @@ pub enum AuxiliaryRequest {
 /// there are two kinds of auxiliary data: bodies and receipts.
 #[derive(Default, Clone)]
 pub struct AuxiliaryData<'a> {
-	/// The full block bytes, including the header.
-	pub bytes: Option<&'a [u8]>,
```

Comment:
I think this might be the most controversial change in this PR. I don't really understand what the role of `AuxiliaryData` is, but the `bytes` member is never used by any of the `Engine`s we have in the code base today. It must be some kind of historical reason for this I am not aware of. Removing the `bytes` member leads to further refactorings e.g. the `AuxiliaryRequest` enum should be simplified etc, but before I proceed with that I'd like to figure out what this stuff was and why it was there.

Reply:
AFAIK,  `AuxiliaryData` was used/intended on `epoch changes` in the light client to fetch `block bodies` and `receipts`. But I guess you mean that the default trait implementation of `Machine` was used and always returned `EpochChange::No`? 

/cc @rphmeier thoughts?

Reply:
I did the work to complete the removal of the `Auxiliary*` stuff [here](https://github.com/OpenEthereum/open-ethereum/pull/11533).

Reply:
Yeah, I'm fine with removing. It was a pre-substrate step towards generalizing over different chains.